pkgs/development/python-modules/ckcc-protocol/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv
+, buildPythonPackage
+, click
+, ecdsa
+, hidapi
+, lib
+, fetchPypi
+, pytest
+, pyaes
+, pythonAtLeast
+}:
+
+buildPythonPackage rec {
+  pname = "ckcc-protocol";
+  version = "0.8.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mbs9l8qycy50j5lq6az7l5d8i40nb0vmlyhcyax298qp6c1r1gh";
+  };
+
+  checkInputs = [
+    pytest
+  ];
+
+  disabled = ! pythonAtLeast "3.6";
```

i would also put this below version
```suggestion
  disabled = pythonOlder "3.6";
```

---

Thanks, good suggestion. Done.